OpenOversight/app/templates/officer.html

```diff
@@ -70,6 +70,18 @@ <h3>General Information
                 <td><b>First Employment Date</b></td>
                 <td>{{ officer.employment_date }}</td>
               </tr>
+              {% if officer.last_employment_date %}
+                <tr>
+                  <td><b>Last Employment Date</b></td>
+                  <td>{{ officer.last_employment_date }}</td>
+                </tr>
+              {% endif %}
+              {% if officer.last_employment_details %}
+                <tr>
+                  <td><b>Last Employment Date Details</b></td>
```

Comment:
Can we change the label that we display to the users to something that is easier understood? Like "Details on Resignation" or "Details on End of Employment" or something? Not exactly sure how this field will be used though